client-participation/webpack.config.js

```diff
@@ -1,27 +1,271 @@
+          test: /\.(handlebars|hbs)$/,
+          exclude: /node_modules/,
+          loader: 'handlebars-loader',
+          options: {
+            ignorePartials: true // We load partials at runtime so ignore at compile-time
+          }
+        },
+        {
+          test: /\.m?js$/,
+          exclude: [
+            /node_modules/
+          ],
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: ['@babel/preset-env', '@babel/react']
+            },
+          },
+        },
+        {
+          test: /\.(png|jpg|gif|svg)$/,
+          exclude: /node_modules/,
+          use: ['file-loader'],
+        },
+        {
+          test: /\.mdx?$/,
+          exclude: /node_modules/,
+          use: ['babel-loader', '@mdx-js/loader']
+        },
+        {
+          test: /\.s[ac]ss$/,
+          exclude: /node_modules/,
+          use: [
+            {
+              loader: 'file-loader',
+              options: { outputPath: 'css/', name: 'polis.css' }
+            },
+            'sass-loader'
+          ]
+        },
+        // Shims for older modules
+        {
+          test: /d3-tip/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: [
+                '@babel/preset-env',
+                '@babel/react'
+              ],
+              sourceType: 'script' // set 'this' to 'window'
+            },
+          },
+        },
+        {
+          test: /deepcopy/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: [
+                '@babel/preset-env',
+                '@babel/react'
+              ],
+              sourceType: 'script', // set 'this' to 'window'
+            },
+          },
+        },
+        {
+          test: /bootstrap\/(transition|button|tooltip|affix|dropdown|collapse|popover|tab|alert)/,
+          use: [
+            {
+              loader: 'imports-loader',
+              options: {
+                imports: [
+                  // Expects jQuery to be present
+                  'default jquery jQuery'
+                ]
+              }
+            }
+          ]
+        },
+        {
+          test: /backbone\/backbone$/,
+          use: [
+            {
+              loader: 'imports-loader',
+              options: {
+                imports: [
+                  // Expects jQuery and lodash to be present
+                  'default jquery $',
+                  'default lodash _'
+                ]
+              }
+            }
+          ]
+        },
+        {
+          test: /handlebones$/,
+          use: [
+            {
+              loader: 'imports-loader',
+              options: {
+                imports: [
+                  // Expects lodash, Backbone and handlebars to be present
+                  'default handlebars Handlebars',
+                  'default backbone Backbone',
+                  'default lodash _'
```

Comment:
I'm taking a look at this. This is a very old module and I'm not sure it can use lodash.

Reply:
https://github.com/pol-is/handlebones/blob/master/handlebones.js

Reply:
Looking at: https://medium.com/paypal-tech/from-require-js-to-webpack-part-2-the-how-1800917fa416

Reply:
https://github.com/walmartlabs/thorax

Reply:
@colinmegill Do you remember why we ended up having to maintain a fork of handlebones?

Reply:
Here's the diff. The polis fork is on the right. 

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/1770265/213101854-81c7112d-78fb-420d-9a8b-b83bec9d4beb.png">

...

<img width="1249" alt="image" src="https://user-images.githubusercontent.com/1770265/213101890-e3e9a9a2-f2df-4443-acfe-db8f98919199.png">

...

<img width="1314" alt="image" src="https://user-images.githubusercontent.com/1770265/213101998-45729edf-77e0-464e-9f05-9850dc1a4fee.png">

I read this change as safety checks after an encountered undefined case.

Reply:
See, especially: 

`if (nextEl)` 

which prevents `cannot read property nextSibling of undefined`